index.html

```diff
@@ -65,6 +65,79 @@ <h2>Introduction</h2>
       TODO(henbos): Fill in.
     </p>
   </section>
+  <section id="*">
+    <h3>
+      <dfn>RTCRtpReceiver</dfn> extensions
+    </h3>
+    <p>
+      Let <code>RTCRtpReceiver</code> objects have an
+      <dfn>[[\ReceiverJitterBufferDelayHint]]</dfn> internal slot initially
+      initialized to <code>null</code>.
+    </p>
+    <pre class="idl">partial interface RTCRtpReceiver {
+  attribute double? jitterBufferDelayHint;
```

Comment:
Depending on how much this is a "hint" and how much this is a "target" we may want to revisit the name. I'm not sure yet.